src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -0,0 +1,200 @@
+import Connector from '../engine/Connector.mjs';
+const EXIF = require('exif-js');
+
+export default class VizShonenJump extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'vizshonenjump';
+        super.label = 'Viz - Shonen Jump';
+        this.tags = [ 'manga', 'english' ];
+        this.url = 'https://www.viz.com/shonenjump';
+        this.requestOptions.headers.set( 'x-referer', this.url );
+        this.config = {
+            /* Since the website requires us to fetch the image data a few seconds after the API call
+            *  we need to ensure we can download the data fast enough.
+            *  Also the website itself only loads a maximum of 4 pages at a time to ensure that.
+            */
+            concurrent: {
```

Comment:
I'm not quite sure, must images be downloaded as fast as possible or must they be throttled to not download as fast as possible. Is it really necessary to introduce throttling on the connector level? The downloader already has throttling support which can be used by configuring the `throttling` property.

Reply:
After issuing the API call we retrieve the image url, but this is only valid fro a few seconds. So if we would retrieve all image urls we require to download them all at once (start immediately). Therefore later pages may start to download too late.
My understanding for `throttling` was that it does not work wit `_handleConnectorURI()`?